validator/lib/validateTileset.js

```diff
@@ -35,6 +35,12 @@ function validateTopLevel(tileset) {
     if (!defined(tileset.geometricError)) {
         return 'Tileset must declare its geometricError as a top-level property.';
     }
+
+    if (defined(tileset.asset.gltfUpAxis)) {
+        if (tileset.asset.gltfUpAxis !== 'X' && tileset.asset.gltfUpAxis !== 'Y' && tileset.asset.gltfUpAxis !== 'Z') {
```

Comment:
Create a variable for `tileset.asset.gltfUpAxis` so it doesn't have to be repeated multiple times.